static/app/stores/groupingStore.tsx

```diff
@@ -328,8 +330,15 @@ const storeConfig: GroupingStoreDefinition = {
         return newItems;
       },
       similar: ([issue, scoreMap]) => {
+        // Check which similarity endpoint is being used
+        const hasSimilarityEmbeddingsFeature = requests[0].endpoint.includes(
```

Comment:
we don't have this typescript feature enabled yet, so you'll need to check that the item exists
```suggestion
        const hasSimilarityEmbeddingsFeature = requests[0]?.endpoint.includes(
```